src/features/FeatureExtractor.js

```diff
@@ -121,6 +133,12 @@ class FeatureExtractor {
             shard,
             userRegion } = { } } = identityEntry;
 
+        if (!this.endpointId) {
+            const { endpointId } = identityEntry;
+
+            this.endpointId = endpointId;
```

Comment:
Are you sure this isn't mixing the endpointID of the rtcstats client with the endpointID of the clients signaling?
I think I saw both being labeled endpointId in the rtcstats server code, which is super confusing. We just should make sure we don't mix them, because that could result in strange data at some point.

Reply:
Thanks for the comment Nils, I checked and it looks good 👍